packages/core/src/renderer/createDOMRenderer.ts

```diff
@@ -1,3 +1,6 @@
+import { injectDevTools } from '../devtools/injectDevTools';
+import { isDevToolsEnabled } from '../devtools/isDevToolsEnabled';
+import { MK_DEBUG } from '../devtools/store';
```

Comment:
Let's create `./devtools/index.ts` that exports _public API_ to be used in other lib files.

```suggestion
import { injectDevTools, isDevToolsEnabled, MK_DEBUG } from '../devtools';
```